CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changes
 
+## NEXT_RELEASE
```

Comment:
We still haven't released 0.19 so you can just include these below (it is building up to quite a release...)